rikaicontent.js

```diff
@@ -405,6 +410,10 @@ var rcxContent = {
 
 		this.keysDown[ev.keyCode] = 1;
 
+		if (!shouldPreventDefault) {
+			return;
+		}
+
 		// don't eat shift if in this mode
 		if (true/*!this.cfg.nopopkeys*/) {

```

Comment:
Should we just roll your new variable into this if statement since it seems like it was meant to be conditional anyway?

Reply:
That sounds good to me. I had avoided modifying the if statement on line 418 as I did not understand the nature of the comment and so decided to leave it be.